bzt/cli.py

```diff
@@ -485,6 +485,14 @@ def apply_overrides(self, options, dest):
 
         dest.dump()
 
+    def __apply_mult_override(self, obj, key, replace_value):
+        for k, v in obj.items():
```

Comment:
Fails if obj is list. Please fix!

Reply:
Hi @corvustristis ,

Made changes to handle list.

Please let me know of any other changes.

Thanks and Regards, Venkatesh